components/Terminal.js

```diff
@@ -7,7 +7,8 @@ import './Terminal.css';
 console.oldLog = console['log']; // eslint-disable-line no-console, dot-notation
 
 function handleLogging(method, addToOutput) {
-  console[method] = (...args) => { // eslint-disable-line no-console
+  console[method] = (...args) => {
+    // eslint-disable-line no-console
```

Comment:
This does not work has to be on the same line. If you really don't like it on the same line you can add a line between 9 and 10 and put `// eslint-disable-next-line`